core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-card/dot-pages-card.component.scss

```diff
@@ -1,5 +1,10 @@
 @use "variables" as *;
 
+:host:hover {
+    box-shadow: $md-shadow-5;
+    transition: box-shadow 0.35s ease-in-out;
+}
```

Comment:
What is this for?

Reply:
This is the hover effect for the panel of a bookmark

Reply:
Ok